models/DataObject/ClassDefinition/Data/Datetime.php

```diff
@@ -483,4 +483,18 @@ protected function doGetDefaultValue($object, $context = [])
 
         return null;
     }
+
+    /**
+     * @param \DateTimeInterface|null $oldValue
+     * @param \DateTimeInterface|null $newValue
+     *
+     * @return bool
+     */
+    public function isEqual($oldValue, $newValue)
+    {
```

Comment:
no seconds ? 

    public function getDataForResource($data, $object = null, $params = [])
    {
        $data = $this->handleDefaultValue($data, $object, $params);

        if ($data) {
            $result = $data->getTimestamp();
            if ($this->getColumnType() == 'datetime') {
                **$result = date('Y-m-d H:i:s', $result);**

Reply:
should be consistent then?
https://github.com/pimcore/pimcore/blob/0588c64d9ed7c9f0ca829cb9a8006ccc6aeff5ba/models/DataObject/ClassDefinition/Data/Datetime.php#L228
https://github.com/pimcore/pimcore/blob/0588c64d9ed7c9f0ca829cb9a8006ccc6aeff5ba/models/DataObject/ClassDefinition/Data/Datetime.php#L248

Reply:
this is just for creating a human readable presentation, we are completely free what we return (as soon as the version comparison is switched to isEqual of course)

Reply:
what about CSV Export? you get incomplete data in export.

Reply:
changed it to be consistent at all places.